existential-vs-universal-quantification/article.md

```diff
@@ -0,0 +1,160 @@
+# Existential vs universal quantification
+
+## Introduction
+From type theory we know about existence of two quantifiers - `$\forall$` and `$\exists$`.
+The former means: `"For every term x the statement is true"`, the later: `"There is at least one term x for which this statement is true"`.
+
+Since Haskell is based on type theory there is a place for these quantifiers in language syntax, however, in Haskell we are talking not about some terms and statements, but about types. 
+By default, all type variables are bounded by `forall` quantifier, which means that every type, that passes constraints, can be used inside this function/type declaration.
+You can't see this quantification without using a specific extension - "Explicit Forall".
+
+## Universal quantification and ExplicitForall
```

Comment:
Perhaps there can be a bit clearer structure. 

## Universal quantification 

(Explain what it is)

### Where can it be used? 

### Example: x 

## Existential quantification 

(Explain what it is)

### Where can it be used? 

### Example: y 

## Some cool places where people can learn more